fuel-android/build.gradle.kts

```diff
@@ -1,7 +1,16 @@
+val kotlin_version: String by extra
 dependencies {
     api(project(":fuel"))
     implementation(Dependencies.kotlinStdlib)
     testImplementation(Dependencies.robolectric)
     testImplementation(Dependencies.junit)
     testCompile(Dependencies.mockServer)
+    compile(kotlinModule("stdlib-jdk7", kotlin_version))
```

Comment:
oh course, import org.jetbrains.kotlin.config.KotlinCompilerVersion 
you can use kotlinModule("stdlib-jdk7", KotlinCompilerVersion.VERSION)